views/homepage.handlebars

```diff
@@ -1,12 +1,29 @@
-<div class="row">
+<ul class="list-group">
+  <a href="#" class="list-group-item list-group-item-action active" aria-current="true">
+    CHOOSE A TRAIL BELOW
+  </a>
+  <li class="list-group-item"> <a href= "/results/1">Angels Landing Trail</a></li>
+  <li class="list-group-item"><a href= "/results/3">Devils Bridge Trail</a></li>
+  <li class="list-group-item"><a href= "/results/2">Emerald Lake Trail</a></li>
+  <li class="list-group-item"><a href= "/results/3">Alum Cave Trail to Mount LeConte</a></li>
+  <li class="list-group-item"><a href= "/results/4">Cathedral Rock Trail</a></li>
+  <li class="list-group-item"><a href= "/results/5">Mount Lafayette and Franconia Ridge Trail Loop</a></li>
+  <li class="list-group-item"><a href= "/results/6">Navajo Loop and Queen's Garden Trail</a></li>
+  <li class="list-group-item"><a href= "/results/7">Lake Blanche Trail</a></li>
+  <li class="list-group-item"><a href="/results/8">Lake 22 Trail</a></li>
+  <li class="list-group-item"><a href="/results/9">Delicate Arch Trail</a></li>
```

Comment:
the /href to these are out of order which is probably why you're getting weird search results